bcl2fastq/index_summary.py

```diff
@@ -45,6 +44,10 @@
                 -  multiple select gives error due to inconsistent values
           """
 layout = html.Div(children=[
+    dcc.ConfirmDialog(
+        id='error',
+        message='You have input an incorrect run. Click "Ok" to return to the most recent run.'
```

Comment:
What happens when you click `Cancel`?

Reply:
Nothing, it does the same thing as if you click 'ok'. There doesn't seem to be a way to only show one button

Reply:
Clarify that in the Message string, so the user knows it doesn't matter which button they click.